pycode/memilio-simulation/memilio/simulation/abm.cpp

```diff
@@ -75,9 +71,8 @@ PYBIND11_MODULE(_simulation_abm, m)
         .def_readwrite("specificity", &mio::abm::TestParameters::specificity);
 
     pymio::bind_Index<mio::abm::AgeGroup>(m, "AgeIndex");
-    pymio::bind_Index<mio::abm::VaccinationState>(m, "VaccinationIndex");
-    pymio::bind_CustomIndexArray<mio::UncertainValue, mio::abm::VirusVariant, mio::abm::AgeGroup,
-                                 mio::abm::VaccinationState>(m, "_AgeVaccinationParameterArray");
+    pymio::bind_CustomIndexArray<mio::UncertainValue, mio::abm::VirusVariant, mio::abm::AgeGroup>(
+        m, "_AgeVaccinationParameterArray");
```

Comment:
With VaccinationState being removed, Vaccinations and related functions should be added to the python code.

Reply:
I add now the definition and function for Vaccination.